### DISCUSSION TITLE
Foreign key with nuxt 3

### DISCUSSION THREAD

**Discussion by vickykeen** [supabase/supabase#15675] [Questions]
Hi guys, 
First of all thank you to the Supabase team for this amazing platform.  Here is the scenario..
I have created two tables one is proposal and another is customers .  In both i have create column name id (int8) as primary key

![image](https://github.com/supabase/supabase/assets/50521644/7275a219-f9b5-4704-bed9-6d36eb64f925)

In proposal i have also created customer_id which is also fk for the customers table as shown below
![image](https://github.com/supabase/supabase/assets/50521644/7decdeb9-e043-4f0d-8866-1b7e7f1a6bc4)

the problem is I am able to create name for both table but when i tried to retrieve the info from customers table through proposals table as foreign key its return null value my code is as below

script

    const { data: proposals } = await useAsyncData('proposals', async () => {
        const { data } = await client.from('proposals').select('id, namel, customers(*)')
        .eq('user', user.value.id).order('created_at')
        return data
    });

in nuxt 3. I am not sure how to do it

**Comment by steve-chavez** [comment]
> table as foreign key its return null 

If it returns `null` instead of an error then it means supabase-js is detecting the relationship. Likely you have an RLS policy filtering the results.

**Reply by vickykeen** [reply]
Hi,
Thanks for the reply Mr Steve but i have Enable read access for all users in RLS still showing the same result ??
